IntelliTect.TestTools.Selenate/GoogleSearch/ExampleTests.cs

```diff
@@ -0,0 +1,57 @@
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using IntelliTect.TestTools.Selenate;
+
+namespace GoogleSearch
+{
+    [TestClass]
+    public class ExampleTests
+    {
+        [TestInitialize]
+        public void Setup()
+        {
+            Browser = new GoogleBrowser(BrowserType.Chrome);
+            Google = new GoogleFunctions(Browser);
+            Harness = new GoogleHarness(Browser);
+        }
+
+        [TestMethod]
+        public void SearchForSeleniumOnGoogle()
+        {
+            Assert.IsTrue(Google.SearchForItem("selenium automation"), 
+                "No search results displayed when they were expected");
+            Assert.IsTrue(Google.FindSearchResultItem("Selenium - Web Browser Automation"),
+                "Did not find a specific search result for Selenium - Web Browser Automation");
+        }
+
+        [TestMethod]
+        public void VerifySeleniumDoesNotExistForElement()
+        {
+            Google.SearchForItem("selenium element");
+            Assert.IsFalse(Google.FindSearchResultItem("Selenium - Web Browser Automation"),
+                "Found a specific search result for Selenium - Web Browser Automation when none was expected");
+        }
+
+        [TestMethod]
+        public void ReturnToHomepage()
+        {
+            Google.SearchForItem("selenium automation");
+            Google.GoToHomePage();
+            Assert.IsFalse(Browser.IsElementDisplayedAndExisting(Harness.SearchResultsDiv.By),
+                "Search results displayed when they were not expected");
+        }
+
+        [TestCleanup]
+        public void Teardown()
+        {
+            Browser.TakeScreenshot();
+            Browser.Driver.Quit();
+            Browser = null;
```

Comment:
Are these null sets needed? Won't the GC simply clean these up for you?

Reply:
Oops! Yes, it will clean those up. Was still in the mindset of the solution I pulled this out of, which was using DI to persist data across test runs. :)